Automattic/jetpack
# Components / My Jetpack: update Spinner component

This PR stops using the Jetpack Spinner component by the core one. It updates the ActionButton of the `Components` pkg, and the Checkout button of My Jetpack.

Fixes https://github.com/Automattic/jetpack/issues/23115

#### Changes proposed in this Pull Request:

* Components / My Jetpack: update Spinner component

#### Jetpack product discussion



#### Does this pull request change what data or activity we track or use?



#### Testing instructions:






* Using Storybook
* Confirm you see the core Spinner in the following `JS PACKAGES / Components / Action Button` story

before | after
------|------
<img width="537" alt="image" src="https://user-images.githubusercontent.com/77539/156586473-9225e601-84c6-4947-8835-f8b6dbc9639e.png"> | <img width="539" alt="image" src="https://user-images.githubusercontent.com/77539/156585823-c00f8a7f-3b07-42ce-88c5-f00643ff63c9.png">

Confirm you see the `Checkout button` ok when it's loading.

https://user-images.githubusercontent.com/77539/156586873-288d46a0-2d95-4f68-b3f1-48b411e4ca83.mov
